frontend/src/components/common/SimpleTable.tsx

```diff
@@ -338,7 +338,7 @@ const useStyles = makeStyles(theme => ({
 
 export interface NameValueTableRow {
   name: string | JSX.Element;
-  value?: string | JSX.Element;
+  value?: string | JSX.Element | JSX.Element[];
```

Comment:
You don't need this if you wrap the whole value in a fragment or another element, right?
It's not a big deal but if we don't need it now, we shouldn't add it.